freeglut/freeglut/src/fg_internal.h

```diff
@@ -548,6 +604,47 @@ do                                                             \
     ((window).CallBacks[WCB_ ## cbname])
 
 /*
+ * FETCH_USER_DATA_WCB() is used as:
```

Comment:
there is a second slightly different version of this comment right below, accident?